src/api/app/controllers/webui/requests_listing_controller.rb

```diff
@@ -18,12 +20,19 @@ def index
 
   private
 
+  # Initialize shared attributes
+  def assign_attributes
+    @url = requests_path
+  end
+
   def filter_requests
-    @bs_requests = filter_by_text(params[:requests_search_text])
```

Comment:
Here we are removing the filter by text. I don't think we should do that...

Reply:
I've moved filter by text `search_for_ids` to query class.

Reply:
You also removed the check to TEXT_SEARCH_MAX_RESULTS... Any reason?

Reply:
There was no specific reason. I initially thought it would provide concise results and there will be no need of the limit, but that wasn't the case, so I've reverted it.